components/clarity-repl/Cargo.toml

```diff
@@ -34,6 +38,16 @@ getrandom = { version = "0.2.3", features = ["js"] }
 rand_pcg = "0.3.1"
 rand_seeder = "0.2.2"
 atty = "0.2.14"
+clarity = { git = "https://github.com/stacks-network/stacks-blockchain.git", branch = "refactor/clarity-lib-merge" }
+
+rusqlite = { version = "=0.24.2", features = [
```

Comment:
As we discussed, it would be nice to get rid of this. The type is only needed to satisfy the `ClarityBackingStore` trait:

datastore.rs:169
```rust
    fn get_side_store(&mut self) -> &Connection {
        panic!("DataStore has no side store")
    }
```

Reply:
Indeed. Should we create an issue to track this?

Reply:
I opened #514